docs/embedded.md

```diff
@@ -1,38 +1,33 @@
 # Embedded C#/WinRT Support
 
-## Overview
-
-**Note**: Embedded support is currently **in preview**. 
-
-Embedded support is a C#/WinRT feature that allows you to compile the c# projection for the component, within the library/app itself. 
+**Note**: Embedded support is currently **in preview** in C#/WinRT v1.4.1.
 
-This is done by setting the build property `CsWinRTEmbedded` to `true` in the app or library's project file. 
+## Overview
 
-Enabling embedded support compiles the sources for WinRT.Runtime with the project.
-For any types from the component which you want to embed in your project, they must be specified using `CsWinRTIncludes`.
+Embedded support is a C#/WinRT feature that allows you to compile C# runtime (WinRT.Runtime) and projection sources into your library or app's output. This is done by setting the build property `CsWinRTEmbedded` to `true` in the app or library's project file. You must specify the types in your component or app which you want to embed using the `CsWinRTIncludes` property.

```

Comment:
WinRT runtime for C# maybe rather than C# runtime?

Reply:
fixed